Userland/Applications/Browser/StorageWidget.cpp

```diff
@@ -8,17 +8,21 @@
 #include "StorageWidget.h"
 #include "CookiesModel.h"
 #include "StorageModel.h"
-#include <Applications/Browser/StorageWidgetGML.h>
-#include <LibGUI/Menu.h>
 #include <LibGUI/TabWidget.h>
 #include <LibGUI/TableView.h>
 #include <LibWeb/Cookie/Cookie.h>
 
 namespace Browser {
 
-StorageWidget::StorageWidget()
+ErrorOr<NonnullRefPtr<StorageWidget>> StorageWidget::create()
+{
+    auto widget = TRY(try_create());
+    TRY(widget->setup());
+    return widget;
```

Comment:
Please do not invent any new ways of running fallible initialization (or any kind of initialization). The GML compiler has [a set method signature](https://github.com/SerenityOS/serenity/blob/master/Base/usr/share/man/man5/GML/Usage.md#initialize-pattern) that gets invoked automatically if available.